qiskit/transpiler/preset_passmanagers/generate_preset_pass_manager.py

```diff
@@ -27,7 +27,7 @@
 from qiskit.transpiler.instruction_durations import InstructionDurations
 from qiskit.transpiler.layout import Layout
 from qiskit.transpiler.passmanager_config import PassManagerConfig
-from qiskit.transpiler.target import Target, target_to_backend_properties
+from qiskit.transpiler.target import Target
```

Comment:
Are we planning to remove `target_to_backend_properties` in the PR removing the class?